### PR TITLE
Simplify dictutil.combineto and add test.

### PR DESCRIPTION
**Pull request by drmingdrmer**
- Fix: tests should use absent kwargs instead of `k=None`.
- add test of zero for absent key: addto({}, {'a': 2}) --> {'a': 2}

### CODE REVIEW

File: dictutil/dictutil.py [hunk 1]
```diff
@@ -360,30 +360,32 @@ def _attrdict(attrdict_clz, d, ref):
...
-
         if k not in a:
-            a[k] = copy.deepcopy(v)
-            continue
-
-        if isinstance(a[k], dict):
-            combineto(a[k], v, op, exclude=sub_exclude, recursive=recursive)
-            continue
+            # use the default constructor of `vb` to get a `zero` value.
+            va = type(vb)()
```
Comment (13yu) [inline dictutil/dictutil.py]: 直接设置为0有时候好像也不太符合预期，如  a: {},  b: {'count': 100}, 可能会希望结果是 {'count': 100}

Comment (drmingdrmer) [reply dictutil/dictutil.py]: 这不就是会给出你要的结果吗...

Comment (drmingdrmer) [reply dictutil/dictutil.py]: 我加了个测试就是你这个场景...

Comment (drmingdrmer) [reply dictutil/dictutil.py]: `addto({}, {'a': 2}) --> {'a': 2}`

Comment (13yu) [reply dictutil/dictutil.py]: 对，我看错了
